include/s1ap/s1apContextManager/s1apContextWrapper_c.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __S1apContextWrappers_C_H
+#define __S1apContextWrappers_C_H
+
```

Comment:
Why we have 2 files with same content.

Reply:
One is a C wrapper. The other one header for the C++ functions.